python/ray/tune/suggest/skopt.py

```diff
@@ -39,19 +54,47 @@ class SkOptSearch(SuggestionAlgorithm):
         >>> }
         >>> algo = SkOptSearch(optimizer,
         >>>     ["width", "height"], max_concurrent=4,
-        >>>     reward_attr="neg_mean_loss")
+        >>>     reward_attr="neg_mean_loss", points_to_evaluate=current_best_params)
     """
 
     def __init__(self,
                  optimizer,
                  parameter_names,
                  max_concurrent=10,
                  reward_attr="episode_reward_mean",
+                 points_to_evaluate=None,
+                 evaluated_rewards=None,
                  **kwargs):
         assert skopt is not None, """skopt must be installed!
             You can install Skopt with the command:
             `pip install scikit-optimize`."""
         assert type(max_concurrent) is int and max_concurrent > 0
+        if points_to_evaluate is None:
+            points_to_evaluate = []
+        elif not isinstance(points_to_evaluate[0], (list, tuple)):
+            points_to_evaluate = [points_to_evaluate]
+        if not isinstance(points_to_evaluate, list):
```

Comment:
Instead of raising ValueErrors, can you use assertions like (assert isinstance(points_to_evaluate, list), err_msg)

Reply:
Actually, raising error is good, except lets use TypeError